controllers/mysqlcluster_controller_test.go

```diff
@@ -664,6 +709,110 @@ var _ = Describe("MySQLCluster controller", func() {
 			Expect(isUpdated).Should(BeFalse())
 		})
 
+		It("should merged with user-defined fluent-bit container spec", func() {
```

Comment:
Please include a test to confirm the config volume can be overwritten by the user-defined configmap in this test (like the example you wrote).

Reply:
I withdrew the above comment. 🙇